.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contributing
+
+Everyone is welcome to contribute to this specification.
+
+Any simple editorial contribution can simply be done with a GitHub Pull Request.
+You can even do an inline edit of the file on GitHub.
+
+Note: Contributions to this repository are intended to become part of Recommendation-track documents governed by the
+[W3C Patent Policy](https://www.w3.org/Consortium/Patent-Policy-20040205/) and
+[Software and Document License](https://www.w3.org/Consortium/Legal/copyright-software). To make substantive contributions to specifications, you must either participate
+in the relevant W3C Working Group or make a non-member patent licensing commitment.
+
+## Style guide to contributors
+
+- the spec uses [ReSpec](https://www.w3.org/respec/)
+- the spec is tidied using [HTML5 Tidy](https://github.com/w3c/tidy-html5). For
+  instructions on running HTML5 tidy, see below.
+- Running tidy is optional. We run tidy on the server after every commit.
+- Put comments in front of sections, for better readability with
+  syntax coloring editors.
+
+## Commit message guidelines
+
+  - When relevant, include `closes #` and the issue number.
+  - The commit message should be short and concise.
+  - Prefix the commit message with one of the following:
+    - `Editorial:` -for editorial/non-normative change to the specification.
+    - `Chore:` - when fixing ReSpec stuff or other non-spec related stuff (e.g., CI).
+    - Normative changes should not be prefixed with anything. These go into the changelog.
+
+## Running HTML5 Tidy
+
+Please make sure you have [HTML5 Tidy](https://github.com/w3c/tidy-html5) installed.
+Don't use the the one that ships with \*nix systems.
+
+You can confirm check the version by running:
+
+```bash
+tidy --version  #HTML Tidy for HTML5 (experimental) for ...
```

Comment:
Mine say `HTML Tidy for Apple macOS version 5.8.0`. Is there a minimum supported version folks should be aiming for?

(the "old" one reports `HTML Tidy for Mac OS X released on 31 October 2006 - Apple Inc. build 1045`

Reply:
I’ll need to check what the latest is. But generally latest is what people should be on. It’s only gets an update like once a year.

Reply:
Got it - the `#HTML Tidy for HTML5 (experimental) for ...` comment made me think my version wasn't correct. But if we just write "the latest version" it'll be fine 99% of the time.